handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -231,6 +220,8 @@ public Boolean run() {
             }
 
             addIfSupported(availableJavaCipherSuites, defaultCiphers, DEFAULT_CIPHER_SUITES);
+            addIfSupported(availableJavaCipherSuites, defaultCiphers, TLSV13_CIPHER_SUITES);
```

Comment:
Can we merge `DEFAULT_CIPHER_SUITES` and `TLSV13_CIPHER_SUITES`? They are added only when supported anyway.

Reply:
nope as we use it also in the JDK impl etc. So lets keep it as it is